java/client/src/main/java/glide/api/commands/StreamBaseCommands.java

```diff
@@ -73,4 +73,19 @@ public interface StreamBaseCommands {
      * }</pre>
      */
     CompletableFuture<Long> xtrim(String key, StreamTrimOptions options);
+
+    /**
+     * Returns the length, or number of entries in the stream stored at <code>key</code>.
```

Comment:
What is the difference between "length" and "number of entries"?

Reply:
If it are same, please simplify the doc.

Reply:
Document uses "number of entries". So, we should remove "the length".